docs/_docs/code-snippets/java/src/main/java/org/apache/ignite/snippets/ConfiguringMetrics.java

```diff
@@ -130,6 +133,19 @@ void jmxExporter() {
         Ignition.start(cfg).close();
     }
 
+    @Test
+    void disableDefaultJmxExporter() {
+
+        //tag::disable-default-jmx-exporter[]
+        IgniteConfiguration cfg = new IgniteConfiguration();
+
+        // Disable default JMX metrics exporter.
+        cfg.setMetricExporterSpi(new MetricExporterSpi[0]);
```

Comment:
JMX exporter will be disabled with System property `IGNITE_MBEANS_DISABLED` or if some other exporter passed. Please, fix this test accordingly.